lectures/pairs_trading/pairs.py

```diff
@@ -36,17 +35,21 @@ def handle_data(context, data):
 def check_pair_status(context, data):
     if get_open_orders():
         return
-    
-    prices = history(35, '1d', 'price').iloc[-context.lookback::]
-    
+
     new_spreads = np.ndarray((context.num_pairs, 1))
-    
+
     for i in range(context.num_pairs):
 
         (stock_y, stock_x) = context.stock_pairs[i]
 
-        Y = prices[stock_y]
-        X = prices[stock_x]
+
+       ***THE CHANGE***
```

Comment:
This change is good, could we remove the comments indicating that it has been changed?